### ISSUE TITLE
Establish cooperation with Safe technical decision-makers

### ISSUE THREAD

**Issue by sergfeldman** [ubiquity/partnerships#57]
### Intro 

This issue was approved at the research meeting. 

---

### Goal

Alex told us that DevPool target audience is technical decision-makers in established Open Source projects. 

For example, Stefan George is a CTO at Gnosis/Safe, so we can be interested in direct cooperation with him.
https://pt.linkedin.com/in/stefangeorge 
https://twitter.com/stefandgeorge

**The area of cooperation: test the DevPool services by solving Safe`s open issues.** 

The specific terms should be discussed. The options:
1. Safe covers 100% of the cost of bounty hunters' payouts. No fees for the DevPool itself. 

2. Ubiquity provides a grant for Safe to partially cover costs. 
Thus, the cost of solving Safe`s issues will be partly from Safe funds, and partly from Ubiquity funds (for example, 90%-10%).
No fees for the DevPool itself. 

3. Other options are welcomed. 

---

### Activity

Establish cooperation with Safe technical decision-makers

- [ ] **Get introduced by the Encode Club**

> Encode Club had 2 AMAs with Stefan George
2022
https://medium.com/encode-club/stefan-george-co-founder-and-cto-of-gnosis-ama-881266c938e7
June 2023
https://www.youtube.com/watch?v=xG9fWAZfGSM
We had a connection with the Encode Club leader Laura.
We can ask her for a DevPool demo for Stefan George.

- [ ] **Participate in the Safe events and ask for an introduction**

> For example, Core Devs calls
https://www.gnosis.io/blog/gnosis-core-devs-call-notes-may-17-2023
Other events
https://coindar.org/en/coin/gnosis-gno

- [ ] **Find other ways to reach out to Safe technical decision-makers.**

> For example, review Discord 
https://discord.com/invite/gnosischain

**Comment by Hodlatoor** [comment]
@sergfeldman 
Good outline! With the target being decision makers, I am wondering if we should have a different "pitch" / intro page for them. Currently we have:
https://dao.ubq.fi/the-devpool

Executives/ higherups typically like to see how a product improves their key metrics like bottom line, turn-around for projects, stats on savings and usability (strategic impact)
Project managers and developers have different concerns. How they can control budget, earn, pay, production cycle (applied impact).

Do we think what we have is sufficient for these two types of actors at https://dao.ubq.fi/the-devpool

**Comment by sergfeldman** [comment]
> @sergfeldman Good outline! 

@Hodlatoor Thank you! It is pleasant to get positive feedback :)

> With the target being decision makers, I am wondering if we should have a different "pitch" / intro page for them. 

On the research meeting this week we decided to create a dedicated pitchdeck for DevPool.
You are welcome to review and add you ideas.
- https://github.com/ubiquity/business-development/issues/58

**Comment by Hodlatoor** [comment]
@sergfeldman @pavlovcik I can take this up: https://github.com/ubiquity/business-development/issues/60

**Comment by 0x4007** [comment]
> I am wondering if we should have a different "pitch" / intro page for them. Currently we have:
https://dao.ubq.fi/the-devpool

- We are to make a "normie" landing page to explain the DevPool, with a call-to-action that says "read the technical docs" which goes straight to our existing documentation e.g. dao.ubq.fi/devpool 
- Pitchdeck is great for sales collateral to our team who will be handling outreach. I believe that the pitchdeck will facilitate sales conversations and allow them to focus on only what we need for them to focus on during the call. Afterwards, I imagine that our counterparties will have sufficient context by skimming through the normie/technical pages (depending on if they are technical.)

The vision is for the research team to do all the thinking/planning/producing sales collateral etc and have the bizdevs focus entirely on execution/repetition (sales is extremely repetitive in my experience.)

**Comment by Hodlatoor** [comment]
> > I am wondering if we should have a different "pitch" / intro page for them. Currently we have:
> > https://dao.ubq.fi/the-devpool
> 
> * We are to make a "normie" landing page to explain the DevPool, with a call-to-action that says "read the technical docs" which goes straight to our existing documentation e.g. dao.ubq.fi/devpool
> * Pitchdeck is great for sales collateral to our team who will be handling outreach. I believe that the pitchdeck will facilitate sales conversations and allow them to focus on only what we need for them to focus on during the call. Afterwards, I imagine that our counterparties will have sufficient context by skimming through the normie/technical pages (depending on if they are technical.)
> 
> The vision is for the research team to do all the thinking/planning/producing sales collateral etc and have the bizdevs focus entirely on execution/repetition (sales is extremely repetitive in my experience.)

Agreed.  I think it would be optimal to have a friendly front that "normies" can get behind and understand and then add links to more technical aspects along the tour.  If a 6 year old gets it, then we explained things properly.
